index.js

```diff
@@ -20,7 +20,6 @@
 var forEach = require('for-each');
 var warning = require('warning');
 var has = require('has');
-var trim = require('string.prototype.trim');
```

Comment:
instead, try `require('string.prototype.trim/polyfill')()`?

Reply:
Not sure what that would solve? It will still end up requiring the polyfil and it's dependencies.

Reply:
it will result in the bundle size being slightly smaller, try it.